geoh5py/ui_json/validation.py

```diff
@@ -116,16 +115,21 @@ def _required_validators(validations):
 
     @staticmethod
     def infer_validations(ui_json: dict[str, Any]):
-        validations = {}
+        """Infer necessary validations from ui json structure."""
+
+        validations: dict[str, dict] = {}
         for key, item in ui_json.items():
             if not isinstance(item, dict):
                 continue
 
             if "isValue" in item:
                 validations[key] = {
-                    "types": [UUID, int, float, Entity],
-                    "association": item["parent"],
+                    "types": [str, UUID, int, float, Entity],
+                    "association": None if item["isValue"] else item["parent"],
```

Comment:
Don't think that's possible. "isValue" means that this can be a dropdown for data, which always requires an association...

Reply:
Yep, changed to only attach association is not isVAlue instead of setting to None